staging/src/k8s.io/csi-translation-lib/plugins/azure_disk.go

```diff
@@ -169,8 +169,8 @@ func (t *azureDiskCSITranslator) TranslateInTreePVToCSI(pv *v1.PersistentVolume)
 		csiSource.ReadOnly = *azureSource.ReadOnly
 	}
 
-	pv.Spec.PersistentVolumeSource.AzureDisk = nil
-	pv.Spec.PersistentVolumeSource.CSI = csiSource
+	pv.Spec.AzureDisk = nil
```

Comment:
`PersistentVolumeSource` is an [embedded struct](https://github.com/kubernetes/kubernetes/blob/04dec17d5b81c77367f4750fa9fb9006069b9156/pkg/apis/core/types.go#L281), so these two are equivalent.